src/mappings/mappingUniquesHandlers.ts

```diff
@@ -1,21 +1,27 @@
 import { SubstrateEvent } from "@subql/types";
 import { UniquesTransfer } from "../types/models";
 
-export async function handleUniquesTransferEvent(event: SubstrateEvent): Promise<void> {
-    logger.debug('uniqueTransferEvent added'  + JSON.stringify(event.toHuman()))
-    const from = event.event.data[0];
-    const to = event.event.data[1];
-    const collectionId = event.event.data[2];
-    const itemId = event.event.data[3];
-    const uniqueTransfer = new UniquesTransfer(`${event.block.block.header.number.toNumber()}-${event.idx}`);
-    const txHash = event.extrinsic.extrinsic.hash.toString();
-    uniqueTransfer.block = event.block.block.header.number.toNumber();
-    uniqueTransfer.txHash = txHash;
-    uniqueTransfer.fromId = from.toString();
-    uniqueTransfer.toId = to.toString();
-    uniqueTransfer.collectionId = collectionId.toString();
-    uniqueTransfer.itemId = itemId.toString();
-    uniqueTransfer.timestamp = new Date(event.extrinsic.block.timestamp).getTime();
+export async function handleUniquesTransferEvent(
+  event: SubstrateEvent
+): Promise<void> {
+  logger.debug("uniqueTransferEvent added" + JSON.stringify(event.toHuman()));
+  const from = event.event.data[0];
+  const to = event.event.data[1];
+  const collectionId = event.event.data[2];
+  const itemId = event.event.data[3];
+  const uniqueTransfer = new UniquesTransfer(
+    `${event.block.block.header.number.toNumber()}-${event.idx}`
+  );
+  const txHash = event.extrinsic.extrinsic.hash.toString();
+  uniqueTransfer.block = event.block.block.header.number.toNumber();
+  uniqueTransfer.txHash = txHash;
+  uniqueTransfer.from = from.toString();
+  uniqueTransfer.to = to.toString();
```

Comment:
True. I removed the relation since the last PR then fromId and toId should be renamed to "to & from" directly. My fault here, thanks